ts/features/fci/store/reducers/fciSignatureRequest.ts

```diff
@@ -94,4 +94,15 @@ export const fciSignatureRequestIdSelector = createSelector(
     pot.isSome(signatureDetailView) ? signatureDetailView.value.id : undefined
 );
 
+/**
+ * Selects the dossier title if present, undefined otherwise.
+ */
+export const fciSignatureRequestDossierTitleSelector = createSelector(
```

Comment:
Do we need to cache this selector? It seems to be a direct read that access an already in-memory data

Reply:
Thanks for the suggestion, we indeed don't need memoization there.